redash/handlers/data_sources.py

```diff
@@ -34,12 +34,15 @@ def get(self):
 
 
 class DataSourceResource(BaseResource):
-    @require_admin
```

Comment:
What do you think about using `@require_permission("list_data_sources")` here instead of `require_access` in the code (keeping serialization logic as implemented here)? I checked how the things work for dashboards and queries and there we have this pattern: require `view_query`/`list_dashboards` both for list and get object handlers - this ensures that the same permissions needed for read access in all cases.